flask_graphql_auth/decorators.py

```diff
@@ -100,6 +124,30 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def query_header_jwt_required(fn):
+    """
+    A decorator to protect a query resolver.
+
+    If you decorate an resolver with this, it will ensure that the requester
+    has a valid access token before allowing the resolver to be called. This
+    does not check the freshness of the access token.
+    """
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        token = _extract_token_value(
```

Comment:
I think "*_header_jwt_required" named decorators should get token from only headers, not in request parameters.